.coveragerc

```diff
@@ -1,3 +1,6 @@
 [run]
 cov-report = term,html
 omit = tests/*
+concurrency = multiprocessing
+parallel = true
+sigterm = true
```

Comment:
suggested by the docs: https://pytest-cov.readthedocs.io/en/latest/subprocess-support.html

but the tests still didn't seem to detect that `error_reporting_wrapper.py`'s `__main__` code did in fact get invoked.